words/predicates.txt

```diff
@@ -15,8 +15,10 @@ abundant
 accessible
 accidental
 accurate
+ace
```

Comment:
Ace is used as a shortened form of asexual - yielding possibilities like "ace-queen" could be risky.